docs/v3/queue.js.html

```diff
@@ -106,7 +106,7 @@ <h1 class="page-title">queue.js</h1>
  * Invoke with `queue.unshift(task, [callback])`.
  * @property {AsyncFunction} pushAsync - the same as `q.push`, except this returns
  * a promise that rejects if an error occurs.
- * @property {AsyncFunction} unshirtAsync - the same as `q.unshift`, except this returns
+ * @property {AsyncFunction} unshiftAsync - the same as `q.unshift`, except this returns
```

Comment:
This is a generated file.  Can you fix this in the source .js file?

Reply:
Oops... done